lib/container_orchestrator/object_definition.rb

```diff
@@ -47,6 +47,27 @@ def deployment_definition(name)
             }
           ]
         end
+      end.tap do |deployment|
+        if File.file?("/etc/pki/ca-trust/source/anchors/root.crt")
```

Comment:
So, this says "if the root cert is available on the orchestrator pod, then set up a volume mount on the child deployments"?  Would it be better to check for presence of `ENV["SSL_SECRET_NAME"]` ?  This way we don't have to call out any specific cert name?  I guess I'm confused because there's a volume mount but _also_ a secret mount?

Reply:
Done